lib/src/definitions.dart

```diff
@@ -165,3 +167,6 @@ typedef OnDirectionsRequestInterceptor = void Function(
     DirectionsRequest directionsRequest);
 typedef OnNavigationRequestInterceptor = void Function(
     NavigationRequest navigationRequest);
+// Directions callback.
+typedef OnDirectionsRequestedCallback = Function(
+    DirectionsMessage directionsMessage);
```

Comment:
Chocame que o callback de directionsRequested este en wyf e os de navigation en sdk
Significa isto que non se van comunicar estados de navegación iniciados dende o mapViewer?

Reply:
Sabía que che chocaría esto, a min tamén me choca pero por ahora é o que hai, mentres non nos metamos no fregado filosófico de "que métodos pertencen ao SDK e cales a WYF". Sin embargo, o que me fixo decidirme é o seguinte:
1) Esta é a callback de petición de ruta (sin navegación). No SDK non existe ningún listener/delegado para esto.
2) O único que fai é pillar a mensaxe do map-viewer de petición de ruta e forwardeala. Si estamos no map-viewer falamos de wyf, non de sdk.

Reply:
Enton as peticions de ruta que inicie o integrador dende o sdk non van chamar a este callback?
Esto diferiria un pouco do caso de navegacion, non?

Reply:
Falado en chamada:
Si, quedaría diferente pero ahora mismo outra cousa que se necesita é o identificador do POI ao que se pide ruta e eso non o temos no SDK (temos Point to, Point from).